lib/matplotlib/tests/test_subplots.py

```diff
@@ -100,6 +100,26 @@ def test_shared():
     check_visible(axs, [False, False, True, True], [True, False, True, False])
 
 
+def test_label_outer_span():
+    fig = plt.figure()
+    gs = fig.add_gridspec(3, 3)
+    # +-+-+-+
```

Comment:
~~~
    # +---+---+---+
    # |   1   |   |
    # +---+---+---+
    # |   |   | 3 |
    # + 2 +---+---+
    # |   | 4 |   |
    # +---+---+---+
~~~

I was a bit confused by the 2 in the condensed form.

Reply:
done